### DISCUSSION TITLE
Next Js direct loading causes 403 forbidden

### DISCUSSION THREAD

**Discussion by KamilGaribov** [vercel/next.js#19416] [Help] [answered]
Hi, Next Js static site generation website routing with <Link> works well.
But direct loading causes error. There is not problem local dev and build mode, only server is problem. What do you think about this bug?
Thank you.

**Comment by KamilGaribov** [comment, marked as answer]
Solved with
module.exports = {
  trailingSlash: true,
};

**Reply by shuhad** [reply]
It doesnt work if i have dynamic route [id].js 
:(

**Reply by ybsat** [reply]
Hi @shuhad, were you able to solve it with the dynamic route? Thank you!

**Reply by GabenGar** [reply]
With static build you have to handle nextjs dynamic routes by your router.
